modules/4337/test/erc4337/ERC4337ModuleNew.spec.ts

```diff
@@ -119,6 +119,37 @@ describe('Safe4337Module - Newly deployed safe', () => {
       expect(await ethers.provider.getBalance(safe.address)).to.be.eq(ethers.parseEther('0'))
     })
 
+    it('should revert when signature length contains additional bytes', async () => {
```

Comment:
This test uses an eoa signature, while the test with the reference entrypoint uses a smart contract signature - why?
Why not both in each of the test files?

Reply:
I thought having a test with Reference entrypoint should be enough. No harm in adding more tests with different signature types as you suggested.
Done here: https://github.com/safe-global/safe-modules/pull/453/commits/22fc197420ae76457903c47b26a72e8e8f40d5c7